azure-iot-device/azure/iot/device/common/pipeline/pipeline_ops_base.py

```diff
@@ -45,7 +46,20 @@ def __init__(self, callback=None):
         self.name = self.__class__.__name__
         self.callback = callback
         self.needs_connection = False
-        self.error = None
+        self.completed = False
+
+    if "pytest" in sys.modules:
```

Comment:
>if "pytest" in sys.modules: [](start = 4, length = 27)

My understanding is that this is only necessary until unittests are fixed?

Reply:
You know what, you're right.  I was using this to make sure everything was changed, and now that all the code is using the new pattern, I'm going to remove this.  

---
In reply to: [333700461](https://github.com/Azure/azure-iot-sdk-python/pull/347#discussion_r333700461) [](ancestors = 333700461)